RecoLocalTracker/SiStripRecHitConverter/python/StripCPEfromTrackAngle_cfi.py

```diff
@@ -1,21 +1,22 @@
 import FWCore.ParameterSet.Config as cms
 
 from RecoLocalTracker.SiStripRecHitConverter.StripCPEESProducer_cfi import *
-StripCPEfromTrackAngleESProducer = stripCPEESProducer.clone()
-StripCPEfromTrackAngleESProducer.ComponentName = cms.string('StripCPEfromTrackAngle')
-StripCPEfromTrackAngleESProducer.ComponentType = cms.string('StripCPEfromTrackAngle')
-StripCPEfromTrackAngleESProducer.parameters    = cms.PSet(
-   mLC_P0         = cms.double(-.326),
-   mLC_P1         = cms.double( .618),
-   mLC_P2         = cms.double( .300),
-   mTIB_P0        = cms.double(-.742),
-   mTIB_P1        = cms.double( .202),
-   mTOB_P0        = cms.double(-1.026),
-   mTOB_P1        = cms.double( .253),
-   mTID_P0        = cms.double(-1.427),
-   mTID_P1        = cms.double( .433),
-   mTEC_P0        = cms.double(-1.885),
-   mTEC_P1        = cms.double( .471),
-   useLegacyError = cms.bool(False),
-   maxChgOneMIP   = cms.double(6000.)
+StripCPEfromTrackAngleESProducer = stripCPEESProducer.clone(
+     ComponentName = 'StripCPEfromTrackAngle',
+     ComponentType = 'StripCPEfromTrackAngle',
+     parameters    = cms.PSet(
+	   mLC_P0         = cms.double(-.326),
```

Comment:
Since you are dropping type specs everywhere else, please also do it here

Reply:
Here the explicit types can not be dropped because they are passed to `cms.PSet()` constructor.

Alternative would be to do
```python
parameters = dict(
    mLC_P0 = -.326
    ...
)
```
but that would leave all existing parameters not specified in `dict()` there, whereas the current `cms.PSet()` effectively removes them.

Reply:
Indeed, thank you @makortel 
Then the same should hold for StripCPEgeometric_cfi.py below...

Reply:
Thanks alot ! I will fix StripCPEgeometric_cfi.py again.